pkg/vpcmodel/abstractVPC.go

```diff
@@ -21,6 +21,9 @@ type NamedResourceIntf interface {
 	UID() string
 	Name() string
 }
+type ZonalNamedResourceIntf interface {
+	ZoneName() string
+}
```

Comment:
might be simpler to add `ZoneName()` to the `NamedResourceIntf` ?

Reply:
not all NamedResourceIntf  has zone, only the one that inherit zonalNamedResource

Reply:
fixed, removed ZonalNamedResourceIntf  and ZonalNamedResource